src/pages/UserAccount/index.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import {
+  Flex, Button, Title, FlexItem, PageSection, Text, TextInput, ButtonVariant, PageSectionVariants, TextVariants,
+} from '@patternfly/react-core';
+import { History } from 'history';
+import React from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import Head from '../../components/Head';
+import { lazyInject } from '../../inversify.config';
+import { GettingStartedTab } from '../../services/helpers/types';
+import { KeycloakAuthService } from '../../services/keycloak/auth';
+import { AppState } from '../../store';
+
+type Props = {
+  history: History;
+} & MappedProps;
+
+type State = {
+  profileUrl: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  login: string;
+}
+
+export class UserAccount extends React.PureComponent<Props, State> {
+
+  @lazyInject(KeycloakAuthService)
+  private readonly keycloakAuth: KeycloakAuthService;
+
+  constructor(props: Props) {
+    super(props);
+
+    const profile = this.props.userProfile.profile;
+
+    const email = profile ? profile.email : '';
+    const login = profile && profile.attributes ? profile.attributes.preferred_username : '';
+    const firstName = profile && profile.attributes ? profile.attributes.firstName : '';
+    const lastName = profile && profile.attributes ? profile.attributes.lastName : '';
+    const profileUrl = KeycloakAuthService.sso ? this.keycloakAuth.getProfileUrl() : '';
+
+    this.state = { login, email, lastName, firstName, profileUrl };
+  }
+
+  private onEdit(): void {
+    const { profileUrl } = this.state;
+    if (profileUrl) {
+      window.location.href = profileUrl;
+    }
+  }
+
+  render(): React.ReactNode {
+    const productName = this.props.branding.data.name;
+    const { firstName, lastName, email, login, profileUrl } = this.state;
+
+    return (
+      <React.Fragment>
+        <Head pageName="User Account" />
+        <PageSection variant={PageSectionVariants.light} isFilled={true}>
+          <Flex direction={{ default: 'column' }}>
+            <FlexItem spacer={{ default: 'spacerSm' }}>
+              <Title headingLevel="h1">Account</Title>
+            </FlexItem>
+            <FlexItem spacer={{ default: 'spacerLg' }}>
+              <Text component={TextVariants.p}>
+                {`This is where you can view and edit your account information for ${productName}.`}
+              </Text>
+            </FlexItem>
+          </Flex>
+          <Flex direction={{ default: 'column' }} style={{ maxWidth: '550px' }}>
+            <Flex direction={{ default: 'column' }}>
```

Comment:
I think using [Form component](https://www.patternfly.org/v4/components/form#limit-width) for all these inputs and button would be better.

Reply:
I have fixed it